src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -375,6 +402,14 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 												</div>
 
 												<div className={styles.currDisplayName}>{token.coin}</div>
+												<div
```

Comment:
Can we make this classname do "cursor: pointer"

Also not necessary b ut would be great if we can add a hover tooltip to explain what exactly the plus means.

Reply:
done.